src/pages/embedded/dev_env/general-integration-consideration.md

```diff
@@ -10,20 +10,34 @@ To make it work in your environment, you have to:
 
  - Include the Luos lib folders in your project compilation;
  - Select the right LuosHAL for your device family in LuosHAL folder, and include `luos_hal.c`, `luos_hal.h` and `luos_hal_config.h` in your project;
- - Change, if necessary, `luos_hal_config.h` in you project, the default configuration created by Luos (before including `luos.h`) in order to match LuosHAL with your hardware (eg: match pins with your design);
+ - Change, if necessary, `luos_hal_config.h` for your project. The default configuration created by Luos is an example for a MCU family that can be modify to fit with your design (eg: match pins with your design);
  - Include `luos.h` on your source file.
 
 ## Configuration
 
-Through the configuration file <a href="https://github.com/Luos-io/Luos/tree/master/Robus/inc/config.h" target="_blank">config.h</a>, Luos allows users to customize many parameters needed for the library. These parameters can be set as compiler variables listed above:
+Luos allow you to configure some parameters allowing to optimize the memory usage to your needs. To make it we advise to use a configuration file call node_config.h. Put the file at the root folder of your project and add it in the compiling variables section of your IDE using a *-include node_config.h*.
+
+- Put all your custom configuration for your project and define your custom Luos service and custom Luos command for your product
+
+- Put all your custom configuration for Luos library to fit with your need
 
 | Parameters | Defaults value | Description |
 | :---: | :---: | :---: |
-| PROTOCOL_REVISION | 0 | Luos protocol version. |
-| DEFAULTBAUDRATE | 1Mbps | Default network bauderate. |
-| MAX_DATA_MSG_SIZE | 128 | Number of bytes of the data field. |
 | NBR_NAK_RETRY | 10 | Number of retries to send after a received NAK. |
 | MAX_CONTAINER_NUMBER | 5 | Number of containers in the node (memory optimisation). |
 | MSG_BUFFER_SIZE | 3*size_msg | Message buffer size. Max size of a message (3 * (7 bytes header + 128 bytes data + 2 bytes CRC)). |
-| MAX_MSG_NB | 2*MAX_CONTAINER_NUMBER | Max number of messages for a container that can be buffered. |
+| MAX_MSG_NB | 2*MAX_CONTAINER_NUMBER | Max number of messages for a container that can be referenced. |
 | NBR_PORT | 2 | Number of PTP on the node ( max 8). See [electronic design](../hardware_topics/electronic-design.md) page.|
+
+You will find the default config for Luos Library in the file <a href="https://github.com/Luos-io/Luos/tree/master/Robus/inc/config.h" target="_blank">config.h</a>,
+
+- Put all your custom configuration for Luos HAL library to modify Luos HAL config and make it fit with your design
```

Comment:
The verb put doesn't seem to be properly used: "put something _somewhere_". The _somewhere_ part is missing in all these sentences, so I suggest either adding it (where do we put the custom configuration?) or replace the verb put by another one like "get" or "use".